pkg/services/tag/tagimpl/sqlx_store.go

```diff
@@ -0,0 +1,34 @@
+package tagimpl
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+
+	"github.com/grafana/grafana/pkg/services/sqlstore/session"
+	"github.com/grafana/grafana/pkg/services/tag"
+)
+
+type sqlxStore struct {
+	sess *session.SessionDB
+}
+
+func (s *sqlxStore) EnsureTagsExist(ctx context.Context, tags []*tag.Tag) ([]*tag.Tag, error) {
+	for _, tagelement := range tags {
```

Comment:
nit, I would suggest using camelCase to become more readable
```suggestion
	for _, tagElement := range tags {
```